pyriemann/utils/mean.py

```diff
@@ -40,7 +41,7 @@ def mean_riemann(covmats, tol=10e-9, maxiter=50, init=None,
     """  # noqa
     # init
     sample_weight = _get_sample_weight(sample_weight, covmats)
-    Nt, Ne, Ne = covmats.shape
+    n_trials, n_channels, n_channels = covmats.shape
```

Comment:
```suggestion
    n_trials, n_channels, _= covmats.shape
```
avoids variable redefinition.

Reply:
Yes, I wondered about that one, but I found it more readable by explicitely indicating that they are spatial covariance matrices. And we are still below the 80-ish characters.

Reply:
Yes, but complete shape is given in the docstring.